package excecao.personalizadaB;

import excecao.Aluno;

public class Causa {

	public static void main(String[] args) {
		
		metodoA(null);

	}

	static void metodoA(Aluno aluno) {
		try {
			metodoB(aluno);
		} catch (Exception e) {
			throw new IllegalArgumentException(causa);
		}
		
	}

	static void metodoB(Aluno aluno){
		if(aluno == null){
			throw new NullPointerException("Aluno est√° NULLLOOO");
		}
		
		System.out.println(aluno.nome);
	}
}